docs/tutorials/pwm_control_flight_controller.rst

```diff
@@ -97,164 +83,204 @@ can be seen in the Information tab, as shown in the image below. For more inform
 
-* **Motor Direction**: This sets what direction the motor considers to be the positive direction for throttle commands from a flight controller. 
-  So in 2D mode, like we are for this example, the motor will always spin this way on a throttle command.  The use of 2D or 3D should match the "FC 2D/3D Mode" parameter, 
-  which defaults to 2D. **Set this to 2D Counter Clockwise**. 
-  
-Any other parameters in this tab can be safely left at their default values.
+The meaning of this parameter is covered in greater detail in the :ref:`throttle_mode_maximums_directions` section of the Feature Reference Manual. 
+
+For this setup, you should **put the module in Voltage mode**. To do that, set the *Mode* parameter to *Voltage* in Control Center, as shown below.
+
+.. figure:: ../_static/tutorial_images/pwm_flight_controller/control_center_mode.png
+    :align: center
+
+    Mode Parameter
+
+Motor Direction
+################
+This sets what direction the module considers to be the positive direction for throttle commands from a flight controller.  When the *Motor Direction* parameter is
+set to either of the 2D directions (*2D clockwise* or *2D counter clockwise*), the module will always spin in the specified direction on a throttle command. For example,
+if the *Motor Direction* was set to *2D counter clockwise*, the module would always spin counter-clockwise in response to a throttle command. Configuring the direction of the
+module and mapping throttle commands to directions is covered in greater detail in the :ref:`manual_throttle` section of the Feature Reference Manual.
+
+For this example, the module should always spin **counter-clockwise**. To configure that, set the *Motor Direction* to *2D counter clockwise* in Control Center, as shown below.
+
+.. figure:: ../_static/tutorial_images/pwm_flight_controller/control_center_direction.png
+    :align: center
 
+    Motor Direction Parameter
+  
 Tuning Tab
 **********
-The Tuning tab has some important parameters as well. Similar to the General tab, the majority of the parameters on this tab are not relevant for this test setup and be can be left at their defaults. The figure
-below provides a quick overview of the correct settings to have on this tab and highlights the important parameters with a red dot. The list below the figure provides greater detail on these important parameters.
+The Tuning tab also has parameters that must be set for this example. Similar to the General tab, the majority of the parameters on this tab are not relevant for this test setup and be can be left at their defaults.
+The sections below provide descriptions of the important parameters in the Tuning tab and details on what they should be set to. They are the same for both DSHOT and PWM setups.
```

Comment:
The sections below describe important Tuning tab parameters as well as their values for the case of this example.